include/boost/detail/winapi/jobs.hpp

```diff
@@ -10,8 +10,23 @@
 
 #include <boost/detail/winapi/basic_types.hpp>
 #include <boost/detail/winapi/access_rights.hpp>
+#include <boost/detail/winapi/detail/cast_ptr.hpp>
 
-#if !defined( BOOST_USE_WINDOWS_H )
+// Microsoft documentation states all of these APIs are available
+// in Windows XP / Windows 2003 or later
+
+#if (BOOST_USE_WINAPI_VERSION >= BOOST_WINAPI_VERSION_WINXP)
+
+#if (BOOST_USE_WINDOWS_H)
```

Comment:
`BOOST_USE_WINDOWS_H` is supposed to be tested for being defined or not.

Reply:
Thanks for catching this.  I will fix it in the next push.